### ISSUE TITLE
Configuration editor does not restore filtered state of unchanged options

### ISSUE THREAD

**Issue by rohitab** [FarGroup/FarManager#577]
### Far Manager version

3.0.6072.0 x64

### OS version

10.0.22623.1037

### Steps to reproduce

1. Run `far:config` to launch the _Configuration editor_
2. Make sure all options, changed and unchanged, are visible
3. Press `Ctrl+H` to show only changed options, the ones marked with an `*`
4. Press `Esc` to close the _Configuration editor_
5. Run `far:config` to launch the _Configuration editor_ again

### Expected behavior

The list of options should still be filtered to only show changed options

### Actual behavior

All options are visible. If you now press `Ctrl+H` nothing happens. You need to press `Ctrl+H` a second time for the list to get filtered.

Ideally, the list of options should be filtered since that was the last state when the _Configuration editor_ was closed. This behavior would match the filtering of _Code pages_ in the Editor/Viewer, and the option to view unmapped volumes in the _Change drive_ menu. If not, then at least `Ctrl+H` should work the first time.

**Comment by alabuzhev** [comment]
Wow, you've found a 10 y.o. bug :)

> This behavior would match the filtering of Code pages in the Editor/Viewer, and the option to view unmapped volumes in the Change drive menu

It looks like I initially wanted it to work like that back in 2012, but never implemented it properly.

The use cases are somewhat different though: you usually filter code pages menu & disks menu to hide unwanted entries permanently, while here it's only to help find something you changed manually (e.g. when something goes wrong).
Now I tend to think that there is no point in hiding config entries permanently.

Ctrl+H should work as expected in 6073.

**Comment by rohitab** [comment]
> Wow, you've found a 10 y.o. bug :)

Yay! :)

I just tested version `3.0.6073.0 x64`. The issue I reported is fixed. Unfortunately, there is a new issue now. Pressing `Ctrl+H` is supposed to _Toggle display of unchanged options_. When you press `Ctrl+H` the first time, the list is filtered to show unchanged options, as expected. However, when you press `Ctrl+H` a second time, nothing happens. It should have toggled the state and restored the complete list of options. This is how it was working is previous versions, and how it's documented. With the current implementation, the only way to view the complete list of options again, is to close and relaunch the _Configuration editor_.

You're right; the use case is a little different. However, maintaining the previous state makes it a little easier to quickly locate options that you are trying out. Let's say you're playing around with `System.Executor.ExcludeCmds`. You make a change, press `Esc` and test out a command. Then you run the _Configuration editor_ again to make changes. If the state was maintained, the modified option would already be visible. Otherwise, you would need to either press `Ctrl+H`, or you would need to search for the option using the right `Alt` key. It's not a big deal though, since it's not used very frequently. Whatever you decide.
